civihr_default_theme/scss/components/_chr-grid.scss

```diff
@@ -0,0 +1,104 @@
+/*
+util classes
+*/
+.chr-separator {
+    background-color: rgb(237, 237, 237);
+    height: 2px;
+}
+
+.chr-table__borderless tbody tr td,
```

Comment:
This appears to be a partials for `.chr_grid` (the file itself should be called `_grid.scss`, there is no need to put the prefix in the file name), so i don't think `.chr_table` should be here

Also, Bootstrap already defines a [`.table`](http://getbootstrap.com/css/#tables) component, so you should expand that (`.table-borderless`, for example, since there is already a `.table-bordered`) instead of creating a new component from scratch